main/Card/index.tsx

```diff
@@ -17,11 +23,27 @@ interface Props {
   testID?: string;
   containerStyle?: ViewStyle;
   children?: ReactNode | ReactNodeArray;
+  image?: ImageSourcePropType;
+  imageStyle?: ImageStyle;
 }
 
 const Card: FC<Props> = (props) => {
-  const { containerStyle, children } = props;
-  return <Container style={[containerStyle]}>{children}</Container>;
+  const { containerStyle, children, image, imageStyle } = props;
+  return (
+    <Container style={[containerStyle]}>
+      {image && (
+        <Image
+          source={image}
+          style={{
+            width: 100,
+            height: 100,
```

Comment:
recommend to replace with styled-component

Reply:
I use styled-component.